components/system/components/GlobalModal.js

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+import * as SVG from "~/components/system/svg";
+import * as Strings from "~/common/strings";
+
+import { css } from "@emotion/react";
+
+const STYLES_BACKGROUND = css`
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(45, 41, 38, 0.6);
+  z-index: ${Constants.zindex.modal};
+`;
+
+const STYLES_MODAL = css`
+  width: 50vw;
+  height: 60vh;
+  position: absolute;
```

Comment:
✂️ no need for position absolute, its in a fixed element container. (slower paint)